lib/challenge.js

```diff
@@ -19,7 +19,15 @@
   // - park
 
 // Answer Starts Here
-
+'Calendar'
+Description: our app uses a calendar to keep track of appointments.
+Every 'calendar' has
+- an appointment name
+- an appointment date
+- an appointment location  (ok, that is three, I tend to use calendars a bit).
+Every 'calendar' can
+- set busy/free status
+- invite another person to the appointment
```

Comment:
At the moment, no. Because... I don't exactly understand the difference between an abstraction and an attribute. Much less so when being testing in a time-limited setting. :)